imports/ui/components/buttons/backButton.js

```diff
@@ -0,0 +1,7 @@
+import './backButton.html'
+
+Template.backButton.events({
+  'click #back-button' () {
+    window.history.back()
```

Comment:
I'm not too familiar w/ `flow-router`, but I looked to see if it had an api to go back to the previous route. I could not find one so I implemented it in this fashion. Open to another approach, though.